src/components/ContactAndReleaseInfo.tsx

```diff
@@ -355,6 +361,12 @@ const ContactAndReleaseInfo: FC<ContactAndReleaseInfoProps> = (props: ContactAnd
         buttonOnClick={() => setShowSuccess(false)}
         showModal={showSuccess}
```

Comment:
Trenger vi å vise success på en modal? de andre 'Lagret' meldingene er bare en liten tekst element på siden. Men dette høres kanskje til en annen sak

Reply:
Tror vi skal ta det med Mona og evt. i en annen sak ja. I første utkast når vi hadde egen redigeringsside var det bare en lagret-tekst, Mona sa at den var for liten og vanskelig å se, så derfor ble det en modal.